smtlib-backends.cabal

```diff
@@ -1,21 +1,30 @@
 name:                smtlib-backends
 version:             0.1
-synopsis:            A library providing low-level functions for SMT-LIB-based interaction with SMT solvers. 
-description:         This library provides an extensible interface for interacting with SMT solvers using SMT-LIB. It has built-in support for running solvers as external processes, and the smtlib-backends-z3 package provides a backend that uses inlined call to Z3's C API.
+synopsis:            Low-level functions for SMT-LIB-based interaction with SMT solvers. 
+description:         This library provides an extensible interface for interacting with SMT solvers using SMT-LIB. It has built-in support for running solvers as external processes, and the smtlib-backends-z3 package provides a backend that uses inlined calls to Z3's C API.
```

Comment:
This line is too long to read.

Reply:
The changelog field will only be useful starting with version `0.2` though, will it not ?

Reply:
Yeah. It would be there only to provide the scaffolding for when it is needed.